### ISSUE TITLE
mos uses deprecated azure cli extension azure-cli-iot-ext

### ISSUE THREAD

**Issue by scarolan** [mongoose-os/mos#44]
Azure command line says that 'azure-iot' is the replacement for 'azure-cli-iot-ext'.

mos is still using the deprecated package.  When I run `az iot hub monitor-events` I get this error about the deprecated package:

```
C:\Users\sean\git_repos\azure-iot-mongoose> az iot hub monitor-events -n seanc-test-iothub
This extension 'azure-cli-iot-ext' is deprecated and scheduled for removal. Please remove and add 'azure-iot' instead.
```

**Comment by incorvia** [comment]
See: https://github.com/mongoose-os/mos/pull/52

**Comment by rojer** [comment]
merged #52
